### DISCUSSION TITLE
Problems with dpgen training

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/dpgen#1485] [Q&A]
I have two questions I would like to ask:
1）For two-dimensional layered materials, if only the bulk structure is trained, can the trained potential function be used to simulate single-layer structures?
2）In dpgen training, my original training dataset is quite large, but after each iteration, new datasets are added and trained together with the original training dataset. However, the size of the new dataset is much smaller compared to the original training dataset. Is there a way to increase the probability of newly added datasets being used for training?

**Comment by njzjz** [comment]
> 1）For two-dimensional layered materials, if only the bulk structure is trained, can the trained potential function be used to simulate single-layer structures?

You can simulate using the dpgen and see how model deviation performs.

> 2）In dpgen training, my original training dataset is quite large, but after each iteration, new datasets are added and trained together with the original training dataset. However, the size of the new dataset is much smaller compared to the original training dataset. Is there a way to increase the probability of newly added datasets being used for training?

[`training_reuse_old_ratio`](https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata/training_reuse_old_ratio) is for such purpose.

**Reply by Edinbur** [reply]
Thank you for your reply.
I am in the process of training for dpgen. For the parameter [training_reuse_old_ratio](https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata/training_reuse_old_ratio), what I read in the manual seems like it can only be activated in init_model mode, where the data is different for each round of iteration, for example, if I have an initial training set of 3500 but after one iteration, there are 200 new datasets added to it, can I turn on the first iteration of the this mode, or is there any other way to specify the probability of the newly added training set being trained?
Thank you very much for taking time out of your busy schedule to respond to my question!
